cps/main.c

```diff
@@ -918,6 +918,13 @@ match_bgp4(struct rte_mbuf *pkt, struct gatekeeper_if *iface)
 	if (ip4hdr->next_proto_id != IPPROTO_TCP)
 		return -ENOENT;
 
+	if (rte_ipv4_frag_pkt_is_fragmented(ip4hdr)) {
+		CPS_LOG(WARNING,
+			"Received IPv4 fragmented packets destined to the Gatekeeper server at %s\n",
```

Comment:
A CPS block may be present in a Grantor server, so the message is misleading.

Please review this and all the other log messages.